recipes/fastparquet/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: fastparquet
+  version: "0.0.1"
+
+source:
+  fn: fastparquet-0.0.1.tar.gz
+  url: https://pypi.io/packages/source/f/fastparquet/fastparquet-0.0.1.tar.gz
+  sha256: "c17b46a6f83314434143ac583968b7b0d99f331c50fae13f82e711b64a926e2a"
+
+build:
+  skip: true  # [win]
```

Comment:
I don't see any indication from the `setup.py` that this should be non-Windows only. I see that `python-snappy` is non-Windows, but from the `README` this seems like an optional requirement.